app/Infrastructure/Console/Commands/SendDailySummaryToSlack.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Toby\Infrastructure\Console\Commands;
+
+use Carbon\CarbonInterface;
+use Illuminate\Console\Command;
+use Illuminate\Support\Carbon;
+use Illuminate\Support\Facades\Http;
+use Spatie\SlashCommand\Attachment;
+use Toby\Domain\DailySummaryRetriever;
+use Toby\Eloquent\Models\Holiday;
+use Toby\Eloquent\Models\User;
+use Toby\Eloquent\Models\Vacation;
+
+class SendDailySummaryToSlack extends Command
+{
+    protected $signature = "toby:slack:daily-summary {--f|force}";
+    protected $description = "Sent daily summary to slack";
+
+    public function handle(DailySummaryRetriever $dailySummaryRetriever): void
+    {
+        $now = Carbon::today();
+
+        if (!$this->option("force") && !$this->shouldHandle($now)) {
+            return;
+        }
+
+        $absences = $dailySummaryRetriever->getAbsences($now)
+            ->map(fn(Vacation $vacation) => $vacation->user->profile->full_name);
+
+        $remoteDays = $dailySummaryRetriever->getRemoteDays($now)
+            ->map(fn(Vacation $vacation) => $vacation->user->profile->full_name);
+
+        $birthdays = $dailySummaryRetriever->getBirthdays($now)
+            ->map(fn(User $user) => $user->profile->full_name);
+
+        $absencesAttachment = Attachment::create()
```

Comment:
These attachments are almost the same as somewhere above. There should be some abstraction for that.